tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp

```diff
@@ -2223,14 +2223,29 @@ static void printSyntacticRenameEdits(sourcekitd_variant_t Info,
     for(unsigned j = 0, je = sourcekitd_variant_array_get_count(Edits);
         j != je; ++j) {
       OS << "  "; // indent
+
       sourcekitd_variant_t Edit = sourcekitd_variant_array_get_value(Edits, j);
+
+      StringRef Path(
+          sourcekitd_variant_dictionary_get_string(Edit, KeyFilePath));
+      if (!Path.empty()) {
+        OS << Path << " ";
+      }
+
       int64_t Line = sourcekitd_variant_dictionary_get_int64(Edit, KeyLine);
       int64_t Column = sourcekitd_variant_dictionary_get_int64(Edit, KeyColumn);
       int64_t EndLine = sourcekitd_variant_dictionary_get_int64(Edit, KeyEndLine);
       int64_t EndColumn = sourcekitd_variant_dictionary_get_int64(Edit, KeyEndColumn);
-      OS << Line << ':' << Column << '-' << EndLine << ':' << EndColumn << " \"";
+      OS << Line << ':' << Column << '-' << EndLine << ':' << EndColumn << " ";
+
+      StringRef BufferName(sourcekitd_variant_dictionary_get_string(Edit, KeyBufferName));
+      if (!BufferName.empty()) {
+        OS << "(" << BufferName << ") ";
+      }
+
       StringRef Text(sourcekitd_variant_dictionary_get_string(Edit, KeyText));
-      OS << Text << "\"\n";
+      OS << "\"" << Text << "\"\n";
```

Comment:
I feel 
```suggestion
      OS << "\"";
      OS.write_escaped(Text);
      OS << "\"\n";
```
outputs more readable results, but I guess that's out of the scope of this PR,

Reply:
I agree, but I didn't want to go and update all the tests 😅